src/xmlrpc.php

```diff
@@ -30,6 +30,12 @@
 /** Include the bootstrap for setting up ClassicPress environment */
 require_once __DIR__ . '/wp-load.php';
 
+if ( get_option( 'disable_xml_rpc', 0 ) === '1' ) {
```

Comment:
This should be Yoda style ideally:
`if ( '1' !== get_option( 'disable_xml_rpc', 0 ) ) {`